agent/listener/psapi.py

```diff
@@ -62,7 +62,7 @@ def make_mountpoint_nodes(partition_name):
             st = os.statvfs(mountpoint)
             iu = st.f_files - st.f_ffree
             iup = 0
-            if iu > 0:
+            if st.f_files > 0:
```

Comment:
This change fixes the issue. I have tested this with the ```nfsd``` filesystem which causes the issue. Tested without ```exclude_fs_types``` containing ```nfsd```.